lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -378,7 +378,8 @@ def test_reset_student_attempts(self):
         reset_student_attempts(self.course_key, self.user, msk, requesting_user=self.user)
         self.assertEqual(json.loads(module().state)['attempts'], 0)
 
-    def test_delete_student_attempts(self):
+    @mock.patch('courseware.module_render.SCORE_CHANGED.send')
+    def test_delete_student_attempts(self, _mock_signal):
```

Comment:
[This test](https://github.com/edx/edx-platform/blob/bcaa873be0af447c520a56cc85e29ea2770fdb6d/lms/djangoapps/instructor/tests/test_enrollment.py#L559) exercises the `SCORE_CHANGED` signal. It actually constructs a proper course and tests that subsection grades are updated when student state is deleted.